answerking_app/tests/test_orderlines.py

```diff
@@ -294,31 +294,3 @@ def test_delete_nonexistant_id_returns_not_found(self):
         # Assert
         self.assertEqual(expected, actual)
         self.assertEqual(response.status_code, 400)
-
```

Comment:
could we put a test in making sure that when a retired item is in a order it is ignored please

Reply:
ok 👍